src/ownership/move-semantics.md

```diff
@@ -21,6 +21,6 @@ fn main() {
 
 * Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
 
-* In Rust, you clones are explicit (by using `clone`).
+* In Rust, primitive types (e.g. `u32`) are copied by default, but to copy structs and tuples you need to explicitly call `clone()`.
```

Comment:
Specifically, types which implement `Copy`.

Reply:
That is correct. I decided not go deep here because one of the followings slides is fully dedicated to `Copy` and `Clone`. 

I am happy to either remove it, if it is not valuable, or we can add something like - "We will discuss this in depth in one of the following sections"

Reply:
```suggestion
* Mention that primitive types like `u32` (actually types implementing the `Copy` trait) are copied.
* In general, if the move semantic is not what you want, you need to explicitly call `clone()` (as in `let s2 = s1.clone();`) to create a copy. In the following section, we will explain more on copying and cloning.
```